Sources/ClientRuntime/Networking/Http/HttpContext.swift

```diff
@@ -107,69 +101,77 @@ public class HttpContextBuilder {
         return self
     }
 
-    @discardableResult
-    public func withEncoder(value: RequestEncoder) -> HttpContextBuilder {
-        self.attributes.set(key: encoder, value: value)
-        return self
-    }
+//    @discardableResult
+//    public func withAuthScheme(value: AuthScheme) -> HttpContextBuilder {
+//        var authSchemes: Attributes
+//        if self.attributes.contains(key: AttributeKeys.authSchemes) {
+//            authSchemes = self.attributes.get(key: AttributeKeys.authSchemes)!
+//        } else {
+//            authSchemes = Attributes()
+//        }
+//        authSchemes.set(key: AttributeKey<AuthScheme>(name: "\(value.schemeId)"), value: value)
+//        self.attributes.set(key: AttributeKeys.authSchemes, value: authSchemes)
+//        return self
+//    }
```

Comment:
should this be left commented out?

Reply:
Yes, see comment above regarding commented code

Reply:
Removed from PR